src/core/ext/filters/stateful_session_affinity/stateful_session_affinity_service_config_parser.cc

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2022 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include <grpc/support/port_platform.h>
+
+#include "src/core/ext/filters/stateful_session_affinity/stateful_session_affinity_service_config_parser.h"
+
+#include <vector>
+
+#include "absl/types/optional.h"
+
+#include "src/core/lib/channel/channel_args.h"
+
+namespace grpc_core {
+
+const JsonLoaderInterface*
+StatefulSessionAffinityMethodParsedConfig::CookieConfig::JsonLoader(
+    const JsonArgs&) {
+  static const auto* loader = JsonObjectLoader<CookieConfig>()
+                                  .OptionalField("name", &CookieConfig::name)
```

Comment:
If I understand `CookieConfig::JsonPostload` right, this field is not optional. Are there any reasons to declare it Optional here?